engine/src/main/scala/cromwell/engine/workflow/ShadowWorkflowActor.scala

```diff
@@ -96,7 +96,7 @@ object ShadowWorkflowActor {
 class ShadowWorkflowActor(workflowId: WorkflowId,
                           startMode: StartMode,
                           workflowSources: WorkflowSourceFiles,
-                          conf: Config) extends LoggingFSM[ShadowWorkflowActorState, ShadowWorkflowActorData] with ActorLogging{
+                          conf: Config) extends LoggingFSM[ShadowWorkflowActorState, ShadowWorkflowActorData] with ActorLogging {
```

Comment:
In the event of an differing opinions, the decision falls back to the submitter Scott. More importantly, it's a single space character, and only my opinion/preference. The PR has my go for merge as is.